Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -974,44 +974,46 @@ def com_google_fonts_check_061(ttFont):
   id = 'com.google.fonts/check/062'
 )
 def com_google_fonts_check_062(ttFont):
-  """Is GASP table correctly set?"""
+  """Is 'gasp' table set to optimize rendering?
+  
+  Traditionally version 0 'gasp' tables were set
+  so that font sizes below 8 ppem had no grid
+  fitting but did have antialiasing. From 9-16
+  ppem, just grid fitting. And fonts above
+  17ppem had both antialiasing and grid fitting
+  toggled on. The use of accelerated graphics
+  cards and higher resolution screens make this
+  approach obsolete. Microsoft's DirectWrite
+  pushed this even further with much improved
+  rendering built into the OS and apps. In this
+  scenario it makes sense to simply toggle all
+  4 flags ON for all font sizes.
+  """
   try:
     if not isinstance(ttFont["gasp"].gaspRange, dict):
-      yield FAIL, "GASP.gaspRange method value have wrong type."
+      yield FAIL, "'gasp' table has no values."
     else:
       failed = False
       if 0xFFFF not in ttFont["gasp"].gaspRange:
-        yield FAIL, "GASP does not have 0xFFFF gaspRange."
+        yield WARN, "'gasp' table does not have a value for all"
+                    " sizes (gaspRange 0xFFFF). It should be set to 1."
```

Comment:
This one needs parentheses. I will merge this as is and fix it with an additional commit to master.